build.sbt

```diff
@@ -8,7 +8,7 @@ ThisBuild / version      := sys.env.getOrElse("BUILD_VERSION", "dev-SNAPSHOT")
 // parsed by project/Versions.scala, updated by updateDependencies.sh
 
 val cpgVersion        = "1.6.10"
-val joernVersion      = "2.0.323"
+val joernVersion      = "2.0.324"
```

Comment:
Once the comparison report looks positive try with the latest version `2.0.327`

Reply:
@khemrajrathore Were you able to resolve the error?

Reply:
Looks like the latest version of overflowdb is `1.192` did you try with `1.92` or `1.192`?

Reply:
Yes, tried with `192` misspelt while copying